modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-new-submissions/index.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import React, {useState} from 'react';
+
+import LineChart from '../../../common/components/line-chart';
+
+export default function () {
+	const [selectedFilterDate, setSelectedFilterDate] = useState('1');
+
+	const options = [
+		{
+			label: 'This Month',
+			value: '1',
+		},
+		{
+			label: '6 MO',
+			value: '2',
+		},
+		{
+			label: 'YTD',
+			value: '3',
+		},
+	];
+
+	const loadData = [
+		{
+			dataColumns: ['New Application', 13, 1, 2, 3],
+			newApplicationsValue: 2,
+			percentNewApplication: 15,
+			period: 1,
+			periodDate: 'Sep 2022',
+		},
+		{
+			dataColumns: ['New Application', 60, 20, 70, 10, 30, 50],
+			newApplicationsValue: 15,
+			percentNewApplication: -10,
+			period: 2,
+			periodDate: 'Dec 2021 - May 2022',
```

Comment:
For the Six Months ago filter, you should use: 
```suggestion
			periodDate: 'Mar 2022 - Sep 2022',
```

Reply:
Done